packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx

```diff
@@ -58,7 +60,7 @@ export class ChoiceGroupOptionBase extends BaseComponent<IChoiceGroupOptionProps
             onChange={this._onChange.bind(this, this.props)}
             onFocus={this._onFocus.bind(this, this.props)}
             onBlur={this._onBlur.bind(this, this.props)}
-            {...getNativeProps(this.props, inputProperties)}
+            {...nativeProps}
```

Comment:
won't this still overwrite className?

Reply:
see line 45 where i select out the "className"

Reply:
That still doesn't fix it, because it's equivalent to this:

let nativeProps = { className: 'nativeClass' };
let inputProps = { className: css(foo, bar) };
let finalProps = { ...inputProps, ...nativeProps };

Whatever was passed in by nativeProps will overwrite whatever inputProps defined.

Reply:
Me neither, cool! Thanks for the explanation :)